lib/assessment.rb

```diff
@@ -13,28 +13,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    reduce (memo, f) { return memo + f }
```

Comment:
`enum.reduce`. Also, Ruby will implicitly return the last value in a block, so the `return` there is unnecessary.